kolla-ansible/ansible/triliovault_globals.yml

```diff
@@ -14,9 +14,6 @@ triliovault_horizon_image_full: "{{ triliovault_docker_registry }}/trilio/{{ kol
 
 ## Triliovault dmapi_workers count
 ## Default value of dmapi_workers is 2
-## This parameter value used to spawn the number of dmapi processes to handle the incoming api requests.
```

Comment:
Don't remove this description of dmapi_workers field

Reply:
Reverted the mistake.
Added description of the dmapi_workers